notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb

```diff
@@ -35,6 +35,15 @@
     "## Settings"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install nibabel"
+   ]
```

Comment:
Should it be added to that notebook, not data-preparation-ct-scan.ipynb? Nibabel is imported there

Reply:
we do not use data-preparation notebook for docs buiilding. I see only that this notebook filed in builds. It failed during some internal imports from monai

Reply:
I see